tests/conftest.py

```diff
@@ -16,5 +16,5 @@ class PiwikSettings(Testing):
         'password': 'piwik',
         'database': 'piwik_db'
     }
-    # FIXME: this should be taken care of in udata-gouvfr
+    # FIXME: this should be taken care of in udata-front
     RESOURCES_SCHEMAGOUVFR_ENABLED = False
```

Comment:
Can we try without this line while we're at it? I _might_ have done something that fixes that a while ago (use default config for plugins).

Reply:
If I remove this line, tests fail on https://github.com/etalab/udata-front/blob/2d16ea24f89331ad1fac20c4e52f8383228014f4/udata_front/frontend/__init__.py#L104.

Reply:
Okay, forget it then for now then :-) Thanks.